internal/service/servicequotas/find.go

```diff
@@ -85,10 +85,7 @@ func findServiceQuotaByID(ctx context.Context, conn *servicequotas.ServiceQuotas
 	}
 
 	if output.Quota.ErrorReason != nil {
-		return nil, &retry.NotFoundError{
-			Message:     fmt.Sprintf("%s: %s", aws.StringValue(output.Quota.ErrorReason.ErrorCode), aws.StringValue(output.Quota.ErrorReason.ErrorMessage)),
-			LastRequest: input,
-		}
+		return nil, fmt.Errorf("%s: %s", aws.StringValue(output.Quota.ErrorReason.ErrorCode), aws.StringValue(output.Quota.ErrorReason.ErrorMessage))
```

Comment:
In case a retry is ever implemented in the future, consider an if-statement to return `Errorf` only for `DEPENDENCY_ACCESS_DENIED_ERROR`.

Reply:
Thanks for the feedback!

I'd argue with the current lack of retry implementation, it's better just to fail on a temporary server error. Otherwise that scenario can result in erroneous resource creation.